Standards

```diff
@@ -0,0 +1,5 @@
+1. One branch per feature
+2. Explicit comments and docs
+3. Clean formatting (KTFMT) (Camel Case)
+4. Use flows or livedata
+5. https://developer.android.com/kotlin/style-guide
```

Comment:
Let's include: Avoid hard coded strings and use stringResource whenever possible.